sage-ios/SAGE/SAGE/EndSemesterViewController.swift

```diff
@@ -0,0 +1,64 @@
+//
+//  EndSemesterViewController.swift
+//  SAGE
+//
+//  Created by Andrew on 1/1/16.
+//  Copyright © 2016 Cal Blueprint. All rights reserved.
+//
+
+import Foundation
+
+class EndSemesterViewController: UIViewController {
```

Comment:
inherit from the formcontroller class i made in my last pr, it'll do the finish button and call the completeForm method for you